core-web/libs/portlets/dot-content-drive/portlet/project.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "portlet",
+    "$schema": "../../../../node_modules/nx/schemas/project-schema.json",
+    "sourceRoot": "libs/portlets/dot-content-drive/portlet/src",
+    "prefix": "lib",
+    "projectType": "library",
+    "tags": [],
```

Comment:
Jal, please add the tags to the project 
` "tags": ["type:feature", "scope:dotcms-ui", "portlet:content-drive"]`